invenio_app_ils/ui/src/invenio_app_ils/common/api/locations/internalLocation.js

```diff
@@ -11,6 +11,11 @@ const get = async internalLocationPid => {
   return response;
 };
 
+const del = async ilocPid => {
```

Comment:
Perhaps use the full name here
```suggestion
const delete = async ilocPid => {
```

Reply:
`delete` is a reserved word for JavaScript